frontend/src/old-pages/CustomImages/CustomImageDetails.tsx

```diff
@@ -38,6 +39,10 @@ import {ValueWithLabel} from '../../components/ValueWithLabel'
 // Constants
 const customImagesPath = ['app', 'customImages']
 
+function truncate(string: string, limit: number) {
+  return string.length > limit ? string.slice(0, limit) + '...' : string
+}
```

Comment:
suggestion: this should be done with CSS since it's not easy to find a definitive value for the `limit` that is valid for every resolution

Some Cloudscape components provide a `wrapText` property. If everything fails, I'd rather use a different text than the value of the actual link